Dockerfile

```diff
@@ -1,9 +1,11 @@
-FROM heroku/cedar:14
-RUN curl "https://github.com/gliderlabs/herokuish/releases/download/v0.3.32/herokuish_0.3.32_linux_x86_64.tgz" \
+FROM heroku/heroku:16
+RUN apt-get update -qq && apt-get install -yy -q daemontools ruby-dev build-tools
+RUN curl "https://github.com/gliderlabs/herokuish/releases/download/v0.3.35/herokuish_0.3.35_linux_x86_64.tgz" \
```

Comment:
don't make this change yet. we handle that on release

Reply:
damn, forgot to revert this i think

Reply:
0.3.35 is the current version and this lines up with what is in master now. see current diff here: https://github.com/gliderlabs/herokuish/pull/333/files#diff-b67911656ef5d18c4ae36cb6741b7965